packages/react/src/errorboundary.tsx

```diff
@@ -79,6 +80,7 @@ function setCause(error: Error & { cause?: Error }, cause: Error): void {
       seenErrors.set(error, true);
       return recurse(error.cause, cause);
     }
+    cause.message = truncate(cause.message, MAX_VALUE_LENGTH);
```

Comment:
No action required at this time, see my other comment but:

The `maxValueLength` can be configured in the SDK's init options. We should respect that and only fall back to `MAX_VALUE_LENGTH` if the option is not set.